train.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
Is this meant to be an example script? Or the an implementation that is used to train new models?

If it is an example, maybe put it in an `examples/` subdirectory. Otherwise the parameters should be input arguments.

Reply:
Thanks for the feedback! I was thinking to use this script to train new models so I have added the parameters as command line arguments.